hubblestack/payload.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+"""
+Many aspects of the salt payload need to be managed, from the return of
+encrypted keys to general payload dynamics and packaging, these happen
+in here
+"""
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
I think this module is assigned to someone else in wiki.
Either you can remove this, or mail to him/her telling that you have done this.

Reply:
Removed payload as it will be ported by Paul.